lib/post_revisor.rb

```diff
@@ -160,18 +160,14 @@ def self.create_small_action_for_category_change(topic:, user:, old_category:, n
       user,
       I18n.t(
         "topic_category_changed",
-        from: category_name_raw(old_category),
-        to: category_name_raw(new_category),
+        from: "##{old_category.slug_ref}",
+        to: "##{new_category.slug_ref}",
       ),
       post_type: Post.types[:small_action],
       action_code: "category_changed",
     )
   end
 
-  def self.category_name_raw(category)
-    "##{CategoryHashtagDataSource.category_to_hashtag_item(category).ref}"
```

Comment:
These classes should not be used directly; also this was an unnecessary way of doing things since all we want here is the `parent:child` slug format which can be the category's job to generate.